### PR TITLE
Introduction MongoDB Data Access tweaks

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

Introduction MongoDB Data Access tweaks. loosely based on user feedback https://mongodb.slack.com/archives/CG53S529Y/p1675063544523789

### Jira

n/a

### Staged Changes

- [Introduction](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/intro_fixes/introduction)

### Reminder Checklist

You might need to also update some corresponding pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-realm update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/introduction.txt [hunk 1]
```diff
@@ -71,7 +71,7 @@ Simply secure:
 
 Synchronized:
   - Seamlessly sync data across devices with Atlas Device Sync .
-  - Avoid writing complex synchronization logic with the :ref:`Database SDKs <realm-sdks>`.
+  - Avoid writing complex synchronization logic with the :ref:` Realm Database SDKs <realm-sdks>`.
```
Comment (cbush) [inline source/introduction.txt:74]:
```suggestion
  - Avoid writing complex synchronization logic with the :ref:`Realm Database SDKs <realm-sdks>`.
```

File: source/introduction.txt [hunk 2]
```diff
@@ -218,12 +176,20 @@ Traditionally developers query the database on the server and access the server
...
 the flexibility of directly querying a database.
 
-App Services includes the :manual:`Query API </tutorial/query-documents>`,
-a powerful API that lets you access MongoDB Atlas data from the client.
-You can query Atlas like you're using a database driver and App Services enforces 
+App Services and the Realm SDKs include MongoDB Data Access,
+an API that lets you access MongoDB Atlas from the client using
+the :manual:`Query API </tutorial/query-documents>`.
+You can query Atlas like you're using a database driver and App Services enforces
 :ref:`data access rules <intro-web-schema>` rules for all requests.
```
Comment (cbush) [inline source/introduction.txt:183]:
```suggestion
:ref:`data access rules <intro-web-schema>` for all requests.
```

File: source/introduction.txt [hunk 3]
```diff
@@ -218,12 +176,20 @@ Traditionally developers query the database on the server and access the server
...
 
-To learn more about the built-in MongoDB service, see :ref:`data-sources`.
+The following Realm SDKs let you query Atlas with MongoDB Data Access from client apps:
+
+- :ref:`Query MongoDB - Java SDK <java-mongodb-data-access>`
+- :ref:`Query MongoDB - .NET SDK <dotnet-mongodb-data-access>`
+- :ref:`Query MongoDB - Node.js SDK <node-mongodb-data-access>`
+- :ref:`Query MongoDB - React Native SDK <react-native-mongodb-data-access>`
+- :ref:`Query MongoDB - Swift SDK <ios-mongodb-data-access>`
+- :ref:`Query MongoDB - Web SDK <web-mongodb-data-access>`
```
Comment (cbush) [inline source/introduction.txt:192]: Confirming: No Flutter yet?

**Review by cbush — APPROVED** [review]

File: source/introduction.txt (hunk 3 above)
Comment (mongodben) [reply source/introduction.txt:192]: correct, scheduled as part of their post GA epic

**Pull request merged**